### PR TITLE
Sticky note integration

### PR DESCRIPTION
**Pull request by anandjram**
Still ugly, but colorful!

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,180 +1,221 @@
-import './App.css';
-import styled from 'styled-components'
-import React, { useReducer } from 'react'
-import { Activity, activityReducer } from './activity/Activity';
-import { Button } from './components/Theme'
+import "./App.css";
+import styled from "styled-components";
+import React, { useReducer } from "react";
+//import { Activity, activityReducer } from "./activity/Activity";
```
Comment (arjun-aarvalabs) [inline src/App.js:4]: Curious why you had to turn this off ?

File: src/App.js [hunk 2]
```diff
@@ -1,180 +1,221 @@
...
       "bravo@aarvalabs.com": {
         userId: "bravo@aarvalabs.com",
         userName: "Bravo",
-        role: "guest"
-      }
-    }
-  }
-}
+        role: "guest",
+      },
```
Comment (arjun-aarvalabs) [inline src/App.js:75]: Is this some auto formatter that you've installed ?

File: src/App.js (hunk 1 above)
Comment (arjun-aarvalabs) [reply src/App.js:4]: Ah, I see you've just renamed.

File: src/App.js [hunk 3]
```diff
@@ -1,180 +1,221 @@
-import './App.css';
-import styled from 'styled-components'
-import React, { useReducer } from 'react'
-import { Activity, activityReducer } from './activity/Activity';
-import { Button } from './components/Theme'
+import "./App.css";
```
Comment (arjun-aarvalabs) [inline src/App.js:1]: Can you fork this repo ? Don't want to merge real activities into the main sandbox repo.

File: src/App.js (hunk 2 above)
Comment (anandjram) [reply src/App.js:75]: Yeah, it's called Prettier. It's a VSCode extension.

File: src/App.js (hunk 3 above)
Comment (anandjram) [reply src/App.js:1]: You do the forking so it's in your name. I'll submit the PR to the new repo.

File: src/App.js (hunk 2 above)
Comment (santhoshsram) [reply src/App.js:75]: I have been using prettier too. sometimes it does too much prettifying.

**Pull request closed without merging**